cmd/podman/create.go

```diff
@@ -506,6 +516,42 @@ func parseCreateOpts(ctx context.Context, c *cli.Context, runtime *libpod.Runtim
 	}
 
 	if c.IsSet("pod") {
+		// Check if pod exists
+		_, err := runtime.LookupPod(c.String("pod"))
+		if err != nil && errors.Cause(err) != libpod.ErrNoSuchPod {
+			// something bad happened
+			return nil, err
+		}
+		if errors.Cause(err) == libpod.ErrNoSuchPod {
+			// pod does not exist; lets make it
+			var podOptions []libpod.PodCreateOption
+			podOptions = append(podOptions, libpod.WithPodName(c.String("pod")))
+			podOptions = append(podOptions, libpod.WithInfraContainer())
+			podOptions = append(podOptions, libpod.WithPodCgroups())
```

Comment:
Nit: you could do all three of these on the same line.  Is this code copied out of podman pod create?  Should we have a common function so we don't end up with bugs.